tests/test_playbooks/inventory_plugin.yml

```diff
@@ -0,0 +1,117 @@
+---
+- hosts: container
+  vars:
+    foreman_groups:
+      - group_a
+      - group_b
+    foreman_hosts:
+      testhost1.example.com: group_a
+      testhost2.example.com: group_b
+  tasks:
+    - name: create database.yml
+      copy:
+        dest: /tmp/database.yml
+        content: |
+          development:
+            adapter: sqlite3
+            database: db/development.sqlite3
+            pool: 10
+            timeout: 5000
+          production:
+            adapter: sqlite3
+            database: db/production.sqlite3
+            pool: 10
+            timeout: 5000
+
+    - name: fetch Foreman container
+      docker_image:
+        name: quay.io/foreman/foreman:2.0-stable
+        # not setting source for now, this needs Ansible 2.8+
```

Comment:
I think, we should only test that plugin with ansible >= 2.9 anyway.

Reply:
yeah this plugin installed w/ collections would only be used w 2.9+ if I understand correctly

Reply:
Yepp, just the playbook gets currently executed on 2.7+, will fix that ;)

Reply:
fixed